Automattic/wc-calypso-bridge
# Remove calypsoify styles and UI elements

In order to use the new WooCommerce navigation experience, calypsoify needs to be removed.  This PR removes any calypsoify-specific styles and UI elements from the bridge.

**Questions:**
1. Do we need a way to "break out" of the ecommerce plan experience?  Or should changes to alerts, theme setup, etc always remain in effect for ecomm plans?

**Follow-ups:**
1. I think there is a great opportunity to remove the connected pages here and instead rely on the `Screen` class defined in the navigation to check if we're on a WC page.
1. `Support` and `Manage Site` (back button) will both be missing from the navigation and will need to be added in.

### Testing

1. Make sure you follow the readme instructions to set up the ecomm plan view.
1. Make sure no regressions have occurred with non-calypsoify elements.
1. Walk through the new OBW to make sure bridge changes related to the ecomm plan still exist there.
